bullets/bullets.gd

```diff
@@ -1,19 +1,19 @@
 extends Area2D
 
-const SPEED = 10
+const SPEED = 400
 
 # Called when the node enters the scene tree for the first time.
 func _ready ():
-	_do_thing()
+	_clean()
 	
-func _do_thing():
+func _clean():
 	await get_tree().create_timer(10).timeout
 	queue_free()
 
 
 # Called every frame. 'delta' is the elapsed time since the previous frame.
-func _physics_process(delta):
```

Comment:
should be _physics_process since you're controlling the position, using normal _physics can result in buggy behavior

Reply:
Reverted back to _physics_process